update texting for STS changes

YuliiaKovalova opened the pull request:
update texting for STS changes

mairaw commented:
This PR seems to contain a lot of other unrelated past commits. Is this correct?

Could we also include here the updates to docs? Change docs.microsoft.com to learn.microsoft.com and remove any en-us locales from links. Thanks!

YuliiaKovalova commented:
> This PR seems to contain a lot of other unrelated past commits. Is this correct?
> 
> Could we also include here the updates to docs? Change docs.microsoft.com to learn.microsoft.com and remove any en-us locales from links. Thanks!

I have cleaned up the commit history and updated the links.
Please take a look :)

mairaw commented on src/.vscode/launch.json:
```diff
@@ -0,0 +1,22 @@
+//debug bash
```
you probably want to add this .vscode folder to .gitignore?

baronfel replied:
it could be useful, especially if the launch configurations (like launchsettings in VS) are path-agnostic. I'd like to see the hard-coded C:/ value changed to something relative to the repo (via the `{workspaceRoot}` parameter - then anyone that clones the repo will get a nice f5 experience out of the box!

YuliiaKovalova replied:
done

mairaw commented:
Thanks for including that change. LGTM.
